.gitignore

```diff
@@ -1,2 +1,9 @@
+# environment and tools
```

Comment:
I don't think this change is necessary. sorry..!

Reply:
Okey👍 Do you mean the file or some strings in it?

Reply:
it means .gitignore 👍

Reply:
Ok) I removed one commit from PR